akka-http-core/src/test/scala/akka/http/impl/engine/client/ConnectionPoolSpec.scala

```diff
@@ -388,7 +388,7 @@ class ConnectionPoolSpec extends AkkaSpec("""
               case x ⇒ fail(x.toString)
             }.toMat(Sink.fold(0)(_ + _))(Keep.both).run()
 
-        Await.result(idSum, 30.seconds) shouldEqual N * (N + 1) / 2
+        Await.result(idSum, 35.seconds) shouldEqual N * (N + 1) / 2
```

Comment:
> I'm also wondering if it would be possible to get rid of most of the test timeout failures. In most tests I've seen the encoded timeouts are AFAIK not related with correctness, for example request/response rendering.

You mean that the actual length of the timeout is not related to correctness? So, that we could just the increase the values a lot because in the happy case it will succeed in a finite amount of time and it will only take a long time for the failure case?